spyder/utils/help/tutorial.rst

```diff
@@ -450,62 +431,58 @@ Shortcuts for useful functions
   (the selection will update automatically) and confirming by pressing
   ``Enter`` when the right name is identified.
 
-- ``Ctrl+Enter`` executes the current cell (menu entry ``Run > Run
+- ``Ctrl+Enter``\* executes the current cell (menu entry ``Run > Run
   cell``). A cell is defined as the code between two lines which start with
-  the agreed tag ``#%%``.
+  the characters ``#%%``, ``# %%`` or ``# <codecell>``.
 
-- ``Shift+Enter`` executes the current cell and advances the
+- ``Shift+Enter``\* executes the current cell and advances the
   cursor to the next cell (menu entry ``Run > Run cell and
   advance``).
 
   Cells are useful to execute a large file/code segment in smaller
   units. (It is a little bit like a cell in an IPython notebook, in
-  that chunks of code can be run independently.)
+  that chunks of code can be run independently).
```

Comment:
This may be a matter of taste (and is not important), but I'd argue that the previous version ".)" is better than the new one ")." as the parenthesis enclose the extra sentence and the full stop below to the sentence.

Reply:
I would argue that this

```
 Cells are useful to execute a large file/code segment in smaller
  units (It is a little bit like a cell in an IPython notebook, in
  that chunks of code can be run independently).

OR

 Cells are useful to execute a large file/code segment in smaller
  units; It is a little bit like a cell in an IPython notebook, in
  that chunks of code can be run independently.
```

Is better :-p

Reply:
> I'd argue that the previous version ".)" is better than the new one ")."

You're indeed correct at least per AP style (which is what we nominally follow per our style guide) and it was I who was mistaken—for a full sentence inside parenthesis, the period/full stop goes inside instead of outside as normal (I've generally avoided the former in my writing to begin with as a matter of preference, so I wasn't as familiar with that exception as I should have been).

>  Cells are useful to execute a large file/code segment in units (it is a little bit like a cell in an IPython notebook, in that chunks of code can be run independently).

The problem with this syntax is that its a bit of a sentence splice there to just use a parenthesis instead of a "stronger" parenthetical (like the semicolon you used below), since both are complete sentences.

>  Cells are useful to execute a large file/code segment in smaller units; it is a little bit like a cell in an IPython notebook, in that chunks of code can be run independently.

This works better. The flow could still be improved, there's a plural/singular discontinuity, and I prefer the em dash here, so how about:

```rst
Cells are useful to execute large files or long blocks of code in smaller units—
like those in an IPython notebook, each cell can be run independently.
```

Reply:
Ended up just splitting with a period instead of a semicolon or a em dash; otherwise implemented the above. Let me know what you think.